src/components/menu/Spectate.tsx

```diff
@@ -40,10 +40,12 @@ class Spectate extends React.Component<WithTranslation, IState> {
                             <div className="field has-addons">
                                 <div className="control">
                                     <label className="button is-static"><Trans
-                                        i18nKey='spectate.code'>code:</Trans></label>
+                                        i18nKey='spectate.code'>Code:</Trans></label>
                                 </div>
                                 <div className="control ">
-                                    <input id="input-code" type="text" name="code" className="input"/>
+                                    <input id="input-code" type="text" name="code"
+                                           className="input"
+                                           placeholder="Enter draft code"/>
```

Comment:
The placeholder value should be localised.
Suggestion for the German value: "Draft-Code eingeben"